common.blocks/i-bem/i-bem.test.bemhtml/i-bem-test.js

```diff
@@ -47,4 +47,5 @@ suite('i-bem block and others', function() {
   unit('boolean mods bem-core/169', 'boolean-mods');
   unit('mods redefinition bem-core/441', 'redefine-mods');
   unit('mix with same block bem-core/804', 'gh-core-804');
+  unit('skip apply next when entering content', 'skip-apply-next');
```

Comment:
better to rename **skip**-apply-next to **reset**-apply-next everywhere (including filenames)